posts/h5ad_conversion/index.qmd

```diff
@@ -0,0 +1,612 @@
+```
+
+Let's visualize the current cell segmentation colored by cell types and the molecular positions of a few selected genes. For more visualization tricks using `Seurat`, please refer to [earlier post](/posts/seurat-cosmx-basics/index.qmd){target="_blank"} and [Seurat's vignette](https://satijalab.org/seurat/articles/seurat5_spatial_vignette_2){target="_blank"} on image-based spatial data analysis. 
+```{r showSegmentation, message=FALSE, eval = FALSE}
+# specify to show cell boundary 
+SeuratObject::DefaultBoundary(seu.obj1[[slideName]]) <- "segmentation"
+
+Seurat::ImageDimPlot(object = seu.obj1, 
+                     fov = slideName, 
+                     
+                     # column name of cell type in meta.data
+                     group.by = "nb_clus_final", 
+                     
+                     # specify which molecules to plot
+                     molecules = c("Slc17a7", "Gad1", "Plp1"), 
+                     mols.size = 1.5, 
+                     
+                     # fixed aspect ratio and flip xy in ploting
+                     coord.fixed = TRUE, flip_xy = TRUE)
+```
+![](figures/Fig-mBrain-S1-cellType-segmentation.png)
+
+You can zoom in to view a subset of cells by specifying which cells to plots.
+```{r showSubset, message=FALSE, eval = FALSE}
+# change idents to "fov" for cell selection
+SeuratObject::Idents(seu.obj1) <- "fov"
+
+Seurat::ImageDimPlot(object = seu.obj1, 
+                     fov = slideName, 
+                     
+                     # column name of cell type in meta.data
+                     group.by = "nb_clus_final", 
+                     
+                     # a vector of chosen cells, plot cells in chosen fovs
+                     cells = SeuratObject::WhichCells(
+                       seu.obj1, idents = c(72:74, 90:92, 97:99, 114:116)), 
+                     
+                     # crop the plots to area with cells only
+                     crop = TRUE, 
+                     
+                     # fixed aspect ratio and flip xy in ploting
+                     coord.fixed = TRUE, flip_xy = TRUE)
+```
+
+![](figures/Fig-mBrain-S1-cellType-fovZoomed.png)
+
+
+### Case 2: multi-slide per `anndata` object {#sec-perStudy}
+
+Since AtoMx&#8482; SIP keeps the spatial coordinates of each slide in their own spatial coordinate space, one would need to arrange the spatial coordinates of multiple sample sections to avoid overlap in XY space when exporting multiple slides in same study-level `anndata` object. The example code below would arrange all slides along Y axis with some spacing between the slides. 
+
+```{r arrangeSlides, message=FALSE, eval = FALSE}
+# extract the segmentation to separate variable
+imgList <- seu.obj@images
+
+# arrange slides along Y axis, add in spacer which is 0.2x of previous slide's coordinate span in Y direction
+spacerFactor <- 0.2
+global_y_offset <- 0 
+
+spatial_coords <- lapply(imgList, function(img){
```

Comment:
Do you need to also move all the segmentation borders and molecule positions?

Reply:
this only takes the cellular coordinates, not the molecules or cell border. It has to apply to all data if you have more than 1 slide.